static/styles/left-sidebar.scss

```diff
@@ -368,17 +359,22 @@ li.top_left_starred_messages {
 .topic-sidebar-arrow {
     position: absolute;
     display: none;
-}
+    right: 10px;
```

Comment:
This kind of change feels functional and it becomes hard to detect if there are any mistakes. Let's add a commit first to move the `right: 10px` and then do the nesting part.